packages/react-core/src/components/Toolbar/examples/ToolbarGroups.tsx

```diff
@@ -65,43 +59,76 @@ export const ToolbarGroups: React.FunctionComponent = () => {
     <React.Fragment>
       <ToolbarItem>
         <Select
```

Comment:
Similar to an above comment re: adding `onOpenChange` to close the select with keyboard (will apply to all 3 selects in this example)